yorkie/backend/backend.go

```diff
@@ -34,6 +35,8 @@ import (
 	"github.com/yorkie-team/yorkie/yorkie/metrics"
 )
 
+const lruExpireCacheSize = 5000
```

Comment:
I think the readability will improve if we express where the cache is used.
```suggestion
const authWebhookCacheSize = 5000
```